### PR TITLE
{{on}} {{fn}} PR and foreign key attrs blog post

### PR DESCRIPTION
**Pull request by jaredgalanis**
Adds {{on}} {{fn}} PR and foreign key attrs blog post

### CODE REVIEW

File: source/2019-08-09-the-ember-times-issue-110.md [hunk 1]
```diff
@@ -10,25 +10,30 @@ responsive: true
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
+{{on}} & {{fn}} In Octane Guides 🔥,
```
Comment (ijlee2) [inline source/2019-08-09-the-ember-times-issue-110.md]:
```suggestion
{{on}} & {{fn}} in Octane Guides 🔥,
```

File: source/2019-08-09-the-ember-times-issue-110.md [hunk 2]
```diff
@@ -10,25 +10,30 @@ responsive: true
...
+Foreign Key Attrs Blog Post 🔑,
+
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [{{on}} & {{fn}} In Octane Guides 🔥](https://github.com/ember-learn/guides-source/pull/912)
```
Comment (ijlee2) [inline source/2019-08-09-the-ember-times-issue-110.md]:
```suggestion
## [{{on}} & {{fn}} in Octane Guides 🔥](https://github.com/ember-learn/guides-source/pull/912)
```

File: source/2019-08-09-the-ember-times-issue-110.md [hunk 3]
```diff
@@ -10,25 +10,30 @@ responsive: true
...
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+**Many thanks** 👏 to [@NullVoxPopuli](https://github.com/NullVoxPopuli) and all the other contributors that worked on the **awesome** [PR](https://github.com/ember-learn/guides-source/pull/912) updating the preview of the [Ember Octane Guides](https://octane-guides-preview.emberjs.com/release/templates/actions/) section dealing with `@action`, the `{{on}}` modifier & `{{fn}}` helper!
+
+These updates to the Guides relate to event handling, binding of context and partial application of functions, and affect the way that actions are used in Ember Octane.
+
+Checkout the PR [here](https://github.com/ember-learn/guides-source/pull/912), and for more context and background see the [blog post](https://www.pzuraq.com/ember-octane-update-action/) from [@pzuraq](https://github.com/pzuraq).
```
Comment (ijlee2) [inline source/2019-08-09-the-ember-times-issue-110.md]:
```suggestion
Check out the PR [here](https://github.com/ember-learn/guides-source/pull/912). For more context, see the [blog post](https://www.pzuraq.com/ember-octane-update-action/) from [@pzuraq](https://github.com/pzuraq).
```

File: source/2019-08-09-the-ember-times-issue-110.md [hunk 4]
```diff
@@ -10,25 +10,30 @@ responsive: true
...
+## [Foreign Key Attrs Blog Post 🔑](https://davidtang.io/2019/08/02/handling-foreign-key-attributes-in-an-api-with-ember-data.html)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Ever have issues handling foreign key attributes in Ember Data? 
+
+Checkout the **clear and concise** [blog post](https://davidtang.io/2019/08/02/handling-foreign-key-attributes-in-an-api-with-ember-data.html) by [@skaterdav85](https://github.com/skaterdav85) that continues his work on Ember Data.
```
Comment (ijlee2) [inline source/2019-08-09-the-ember-times-issue-110.md]:
```suggestion
Check out the **clear and concise** [blog post](https://davidtang.io/2019/08/02/handling-foreign-key-attributes-in-an-api-with-ember-data.html) by [@skaterdav85](https://github.com/skaterdav85) that continues his work on Ember Data.
```

File: source/2019-08-09-the-ember-times-issue-110.md [hunk 5]
```diff
@@ -10,25 +10,30 @@ responsive: true
...
+
+The post includes good patterns that leverage the use of belongsTo in Ember Data and the necessary setup for serializers and adapters.
```
Comment (ijlee2) [inline source/2019-08-09-the-ember-times-issue-110.md]:
```suggestion
The post includes good patterns that leverage the use of `belongsTo` in Ember Data and the necessary setups for serializers and adapters.
```

File: source/2019-08-09-the-ember-times-issue-110.md [hunk 6]
```diff
@@ -10,25 +10,30 @@ responsive: true
...
+
+For a deeper dive you can also checkout [@skaterdav85](https://github.com/skaterdav85)'s [book on Ember Data](https://leanpub.com/emberdatainthewild) or [other blog posts](https://davidtang.io/2019/07/21/embedded-records-in-ember-data-with-json-api.html).
```
Comment (ijlee2) [inline source/2019-08-09-the-ember-times-issue-110.md]:
```suggestion
For a deeper dive you can also check out [@skaterdav85](https://github.com/skaterdav85)'s [book on Ember Data](https://leanpub.com/emberdatainthewild) and [other blog posts](https://davidtang.io/2019/07/21/embedded-records-in-ember-data-with-json-api.html).
```

**Review by ijlee2 — APPROVED** [review]
Thanks for covering both, @jaredgalanis.

I noticed that our CONTRIBUTING.md file is now failing lint check. Do you want to fix it as a part of your branch? I think we now have to specify the language whenever there is a triple backtick.

**Comment by jaredgalanis** [comment]
> Thanks for covering both, @jaredgalanis.
> 
> I noticed that our CONTRIBUTING.md file is now failing lint check. Do you want to fix it as a part of your branch? I think we now have to specify the language whenever there is a triple backtick.

Thanks for the review @ijlee2 ! Yah, I'll take a look at the lint failure now.

**Pull request merged**